tika-pipes/tika-emitters/tika-emitter-s3/src/main/java/org/apache/tika/pipes/emitter/s3/S3Emitter.java

```diff
@@ -16,33 +16,19 @@
  */
 package org.apache.tika.pipes.emitter.s3;
 
-import static org.apache.tika.config.TikaConfig.mustNotBeEmpty;
```

Comment:
Import order had to be fixed?

Reply:
ah yeah. i'm getting fired up on a new laptop with new everything. so i'm setting up my checkstyle profile now